dbbackup/management/commands/_base.py

```diff
@@ -0,0 +1,18 @@
+from optparse import make_option
+from django.core.management.base import BaseCommand, LabelCommand
+
+
+class BaseDbBackupCommand(LabelCommand):
```

Comment:
AH! Sorry, correct... I just looked at mediabackup not the other two, they use LabelCommand yes... but I don't think they use it the right way.